### ISSUE TITLE
Update to remark@13.0.0

### ISSUE THREAD

**Issue by azu** [textlint/textlint#717]
https://github.com/textlint/textlint/tree/master/packages/%40textlint/markdown-to-ast
use remark.

We need to update remark@13.0.0
https://github.com/remarkjs/remark/releases/tag/13.0.0

markdown-to-ast currently includes some markdown extensions

- yaml front matter
- gfm

**Comment by azu** [comment]
```
notable: references (as in, links [text][id] and images ![alt][id]) are no longer present as such in the syntax tree if they don’t have a corresponding definition ([id]: example.com). The reason for this is that CommonMark requires [text *emphasis start][undefined] emphasis end* to be emphasis.
```

may break some rule.

**Comment by massongit** [comment]
All versions of package `trim` lower than 0.0.3 are vulnerable to Regular Expression Denial of Service (ReDoS): CVE-2020-7753
You need to update to `remark-parse` 9.0.0 to fix this vulnerability because versions of package `remark-parse` lower than 9.0.0 depend on `trim` 0.0.1.

**Comment by azu** [comment]
Thanks for the notice.

remark-parse 6 .... 8 diffs.

- [Release 13.0.0 (micromark) · remarkjs/remark](https://github.com/remarkjs/remark/releases/tag/13.0.0)
  - [remarkjs/remark-gfm: remark plugin to support GFM (autolink literals, strikethrough, tables, tasklists)](https://github.com/remarkjs/remark-gfm)
- [Release remark-parse@8.0.0 · remarkjs/remark](https://github.com/remarkjs/remark/releases/tag/remark-parse%408.0.0)
- [Release remark-parse@7.0.0 · remarkjs/remark](https://github.com/remarkjs/remark/releases/tag/remark-parse%407.0.0)

**Comment by azu** [comment]
Related https://github.com/remarkjs/remark/issues/710

**Comment by azu** [comment]
I've tried updated remark in https://github.com/textlint/textlint/pull/767
Almost work is ok, but https://github.com/remarkjs/remark-gfm/issues/16 is a blocker to update.
We need to resolve it or add a workaround for it.

**Comment by azu** [comment]
I've to released 12.0.0 beta 0 that includes remark-parser updates

```
npm install textlint@beta
```

Please try it. If you found problems, please report it.
